downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md

```diff
@@ -30,6 +30,8 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
 
 ## Backward-incompatible changes
 
+- With the Java 11 to Java 17 migration, some Ballerina interop codes can have broken Java codes and require compatible versions of libraries. Please refer to Java 17 migration guides to update them.
```

Comment:
```suggestion
- With the Java 11 to Java 17 migration, some Ballerina interop codes can have broken Java codes and require compatible versions of the libraries. Please refer to Java 17 migration guides to update them.
```
Incorrect format when giving the reference hyper link. Please fix.
Shouldn't this be a breaking change? @warunalakshitha

Reply:
We decided to refrain from adding Oracle websites as references here since it may not be a legal compliance.

Reply:
@praneesha We have listed this under breaking changes.